backend/src/middlewares/auth.middleware.ts

```diff
@@ -0,0 +1,38 @@
+import { NextFunction, Response } from 'express';
+import { verify } from 'jsonwebtoken';
+import { PrismaClient } from '@prisma/client';
+import { HttpException } from '@exceptions/HttpException';
+import { DataStoredInToken, RequestWithUser } from '@interfaces/auth.interface';
+import * as dotenv from 'dotenv';
```

Comment:
you use `import * as dotenv from 'dotenv'` in multiple places.
Best practice is to create some config.ts file create variables that will get their value from environment variables (and for the ones that is possibale to have some default value).
I can show you an example if you want

Reply:
all instances of this import removed. moved to a relevant config file with type checks with 'envalidid' package.